src/core/diff.ts

```diff
@@ -26,6 +26,7 @@ export class Diff {
     documentation: string | undefined,
     hub: string | undefined,
     token: string | undefined,
+    format: string,
```

Comment:
It is related to 'Flags format has a default value, so it's always defined', right ?